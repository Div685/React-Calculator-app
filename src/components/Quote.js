import React from 'react';
import '../style/Quote.css';

function Quote() {
  return (
    <div className="quote">
      <h1>
        Mathematics is not about numbers, equations, computations, or
        algorithms: it is about understanding. â€“William Paul Thurston
      </h1>
    </div>
  );
}

export default Quote;
