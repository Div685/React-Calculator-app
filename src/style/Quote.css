.quote {
  padding: 14rem 8rem;
  font-family: math;
}

.quote h1 {
  font-size: 28px;
  display: flex;
    justify-content: center;
    text-align: justify;
}