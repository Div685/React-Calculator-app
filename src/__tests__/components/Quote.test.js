import React from 'react';
import renderer from 'react-test-renderer';
import {
  render, screen,
} from '@testing-library/react';
import '@testing-library/jest-dom';

import Quote from '../../components/Quote';

describe('Quote Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});

describe('Test Quote component', () => {
  it('test heading content in Quote component', () => {
    render(<Quote />);
    expect(screen.getByRole('heading')).toHaveTextContent('Mathematics is not about numbers, equations, computations,or algorithms: it is about understanding. â€“William Paul Thurston');
  });
});
