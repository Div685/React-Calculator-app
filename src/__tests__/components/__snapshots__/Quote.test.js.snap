// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component renders correctly 1`] = `[Function]`;
